lib/src/arithmetic/signals/floating_point_logics/floating_point_logic.dart

```diff
@@ -73,13 +77,17 @@ class FloatingPoint extends LogicStructure {
   FloatingPointValuePopulator valuePopulator() => FloatingPointValue.populator(
       exponentWidth: exponent.width,
       mantissaWidth: mantissa.width,
-      explicitJBit: explicitJBit);
+      explicitJBit: explicitJBit,
+      subNormalAsZero: subNormalAsZero);
 
   /// Return true if the J-bit is explicitly represented in the mantissa.
   bool get explicitJBit => _explicitJBit;
-
   late final bool _explicitJBit;
 
+  /// Return true if subnormal numbers are represented as zero.
+  bool get subNormalAsZero => _subNormalAsZero;
+  late final bool _subNormalAsZero;
```

Comment:
why not `final bool subNormalAsZero`?  (same for `explicitJBit`)

Reply:
I think this may have been a remnant of when we were inheriting?